packages/create-atomic/templates/package.json

```diff
@@ -1,24 +1,33 @@
 {
   "name": "{{project}}",
-  "description": "Coveo Atomic Project",
+  "description": "{{titleCase project}} project",
   "version": "0.1.0",
   "scripts": {
     "postinstall": "npm run setup-server && npm run setup-cleanup",
     "setup-server": "node ./scripts/setup-server.js",
     "setup-cleanup": "node ./scripts/clean-up.js",
     "start-server": "node ./scripts/start-server.js",
-    "start-app": "node ./scripts/start-app.js",
     "allocate-port": "node ./scripts/port-allocator.mjs",
-    "start": "npm run allocate-port && concurrently --raw \"npm run start-server\" \"npm run start-app\""
+    "build": "NODE_ENV=production webpack",
+    "start": "npm run allocate-port && concurrently --raw \"npm run start-server\" \"webpack serve\""
```

Comment:
Good for a local dev setup, I'll investigate what it would take for a deployment setup next :)